### DISCUSSION TITLE
Custom json_login authenticator

### DISCUSSION THREAD

**Discussion by sukhoy94** [lexik/LexikJWTAuthenticationBundle#963] [Q&A]
Hello, 
I have some problems with implementing one feature. Basically I want to have one dev-password for all my users in my dev environment.
So I think I should create some custom authenticator in which I should somehow to redefine logic in Symfony\Component\Security\Http\EventListener\CheckCredentialListener, or maybe somehow to change behavior of getPassword() method in my User class.

Is there any standard solutions for this using lexit-jwt ?

**Comment by aynaitlamine** [comment]
@sukhoy94 any chances with that
